### PR TITLE
Improve readme

### PR DESCRIPTION
**Pull request by santib**

### CODE REVIEW

**Review by juanmanuelramallo — APPROVED** [review]
C H E T I S I M O

File: README.md [hunk 1]
```diff
@@ -1,7 +1,42 @@
...
+  - [Chaining scopes](#chaining-scopes)
+- [Development](#development)
+- [Contributing](#contributing)
+- [License](#license)
+- [Code of Conduct](#code-of-conduct)
+- [Credits](#credits)
+
+## Motivation
+
+`ActiveRecord` provides us with an amazing abstraction of the database structure, allowing us to form queries in a simple way. Unfortunately models can grow big for several reasons, one of them being adding a lot of scopes or placing querying logic in methods.
```
Comment (fedeagripa) [inline README.md]: `allowing us to form queries` --> `allowing us to write queries` ?

Comment (fedeagripa) [reply README.md]: `Unfortunately models can grow big` --> `Unfortunately models can grow/get fat` ?

Comment (santib) [inline README.md]:
```suggestion
`ActiveRecord` provides us with an amazing abstraction of the database structure, allowing us to write queries in a simple way. Unfortunately, models can grow large for several reasons, one of them being adding a lot of scopes or placing querying logic in methods.
```

Comment (santib) [reply README.md]: I don't like using the word fat, but I changed it for grow large

**Pull request merged**